src/businesschat.cs

```diff
@@ -0,0 +1,44 @@
+//
+// BusinessChat bindings
+//
+// Authors:
+//     Manuel de la Peña <mandel@microsoft.com>
+//
+// Copyright 2017 Microsoft Inc. All rights reserved.
+//
+using System;
+using System.ComponentModel;
+
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+#if MONOMAC
+using XamCore.AppKit;
+#else
+using XamCore.UIKit;
+#endif
+
+namespace XamCore.BusinessChat {
+	[Mac (10,13, onlyOn64: true), iOS (11,3)]
+#if MONOMAC
+	[BaseType (typeof(NSControl))]
+#else
+	[BaseType (typeof(UIControl))]
+#endif
+	interface BCChatButton {
+		[Mac (10,13,4, onlyOn64: true), iOS (11,3)]
+		[Export ("initWithStyle:")]
+		[DesignatedInitializer]
```

Comment:
if that's the default initializer then the type should have `[DisableDefaultCtor]`